docs/network.md

```diff
@@ -0,0 +1,104 @@
+# Working With Network
+
+![playwright network](https://user-images.githubusercontent.com/746130/79428385-f0264900-7f7a-11ea-86e6-cd03190b8de7.png)
+
+Playwright provides APIs to **monitor** and **modify** network traffic, both HTTP and HTTPS.
+Any requests that page does, including [XHRs](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and
+[fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) requests, can be tracked and modified.
+
+
+> **NOTE** As of playwright v0.13.0, Playwright is not yet capable of tracking websocket messages.
+
+
+## Monitor all network activity in page
+
+```js
+const page = await browser.newPage();
+page.on('request', request => console.log('>>', request.method(), request.url()));
+page.on('response', response => console.log('<<', response.status(), response.url()));
+await page.goto('https://example.com');
+```
+
+#### API reference
```

Comment:
I like the back-references to the API in each section, but it somewhat hard to distinguish between the main examples and the following refs. Maybe we can tweak the formatting a bit (smaller font for the API links or bigger margin before the next section) ?

Reply:
Adding  
```html
<br/> <br/>
```
seems to work well in input.md, let's do the same here.